### PR TITLE
Spotify Clone using HTML, Vanilla CSS

### PR DESCRIPTION
**Pull request by chaanakyaaM**
Closes: #252 
- **Title** : Spotify Clone
- **Name:** : Chaanakyaa Milkuri
- **Idenitfy yourself:** : GSSOC'24

### Describe the add-ons or changes you've made 📃

Created Spotify UI Clone using HTML and Vanilla CSS.

### Screenshots 📷

Screenshot (191)

Screenshot (192)

Screenshot (193)

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Note to reviewers 📄

Images used in this project are provided in the assets folder and other few images are from the internet via image link, if you like me to download all those images and keep them in assets folder , just let me know.

### CODE REVIEW

File: Front-end-Projects/Intermediate/Spotify-Clone/Screenshot 1.webp
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Intermediate/Spotify-Clone/Screenshot 1.webp:1]: i have already told you to change the name of 1 screenshot to `screenshot`

File: Front-end-Projects/Intermediate/Spotify-Clone/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+</p>
+</div>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+
+<img src="screenshot 1.webp">
```
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Intermediate/Spotify-Clone/README.md:48]: image

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Front-end-Projects/Intermediate/Spotify-Clone/README.md (hunk 1 above)
Comment (chaanakyaaM) [reply Front-end-Projects/Intermediate/Spotify-Clone/README.md]: sorry for the confusion, can you kindly review my new changes in the PR.

**Comment by chaanakyaaM** [comment]
Done

**Review by Avdhesh-Varshney — APPROVED** [review]
@chaanakyaaM PR Approved 🎉

**Pull request merged**